tests/infra/storage_mixin.py

```diff
@@ -55,8 +54,7 @@ def upload_file(self: typing.Any, bundle_file: TestFile, replica: Replica) -> st
         response = self.upload_file_wait(
             bundle_file.url,
             replica,
-            file_uuid=bundle_file.uuid,
-            bundle_uuid=bundle_file.bundle.uuid,
+            file_uuid=bundle_file.uuid
```

Comment:
Don't like trailing commas? :)

Reply:
Is the point for them to have minimal diffs when adding removing items?

Reply:
Yup. :)

NBD.